### PR TITLE
Laybuy Payment Icon

### PR DESCRIPTION
**Pull request by Sanjeev-Gupta**
I have added my payment icon "laybuy.svg" .Please merge and make it live asap.

### CODE REVIEW

**Comment by Sanjeev-Gupta** [comment]
Hi,
Please provide details to fix the test failing.

File: db/payment_icons.yml [hunk 1]
```diff
@@ -206,6 +206,12 @@
   name: paysafecard
   label: PaySafeCard
   group: other
+
```
Comment (AnthonyClark) [inline db/payment_icons.yml]: Please remove these empty newlines

**Comment by Sanjeev-Gupta** [comment]
Hi,

I removed the empty newlines on my "laybuy" branch and made a Pull 
Request to activemerchant/paymenticon. Please check and reply if I have 
to do something more.

Thanks

**Comment by Sanjeev-Gupta** [comment]
Hi,

Thanks for your help. But when icon becomes live on shopify ? I checked 
but my logo is still not listing. Please reply asap.

Thanks.

**Comment by AnthonyClark** [comment]
Hey @Sanjeev-Gupta, I can't give a definitive time, most probably in the next week or two.

**Comment by Sanjeev-Gupta** [comment]
Hi,

Please provide me the date when Laybuy Logo be live, earlier you said 
"most probably in the next week or two", but still now Its not live. 
Please provide problem if any and reply soon.

Thanks.

**Comment by Sanjeev-Gupta** [comment]
Hi,

Sorry, Its Live. Thanks to you. Ignore my last Mail.

**Pull request merged**